mcux/README

```diff
@@ -4,13 +4,15 @@ MCUXpresso SDK
 Origin:
    NXP MCUXpresso SDK
    https://github.com/NXPmicro/mcux-sdk
+   SHA: 076dc8823ac7c8c5499688418aa7691160a10ec6
```

Comment:
This commit does not include the LTO fix merged to upstream https://github.com/nxp-mcuxpresso/mcux-sdk/pull/166

Reply:
I have pulled in the latest MCUX-SDK release and this commit is included.

Reply:
@mmahadevan108 Can you update the SHA that's listed? It references a much earlier commit **without** the LTO fix.

Reply:
Thanks for pointing this, I have updated the SHA

Reply:
@mmahadevan108 Thank you! Having the SHA's listed in the `README` is a great addition -- super helpful!